Makefile

```diff
@@ -118,6 +120,9 @@ update-minikube: build build-dummy-ignition
 build-dummy-ignition-image: build-dummy-ignition
 	docker build --network=host -f Dockerfile.ignition-dummy . -t ${DUMMY_IGNITION}
 
+build-livecd:
```

Comment:
Currently the target for building the image is:
```build-assisted-iso-generator-image: build```
This target is used in the CI job for publishing the image (https://github.com/openshift/assisted-service/blob/master/Jenkinsfile#L42)
Please adjust.
Probably need to update the Jenkins file as well

Reply:
I added the target `build-livecd` as a dependency for `build-assisted-iso-generator-image target`. That should be enough.